packages/kit/src/exports/vite/index.js

```diff
@@ -381,14 +378,14 @@ function kit({ svelte_config }) {
 		 */
 		async configureServer(vite) {
 			// set `import { version } from '$app/environment'`
-			(await vite.ssrLoadModule(`${runtime_prefix}/env.js`)).set_version(
+			(await vite.ssrLoadModule(`/@fs${runtime_prefix}/shared.js`)).set_version(
```

Comment:
note to self: this probably doesn't need the `/@fs` prefix (was trying to get windows stuff to work), and also it makes no sense that we're importing this module twice, once with `runtime_prefix` and once with `runtime_base` (I can't remember what the difference is)